### PR TITLE
Added File Splitting Capabilities

### PR DESCRIPTION
**Pull request by homemmakako**

### CODE REVIEW

File: src/configuration/path-alias-utils.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+export class PathUtils {
+  public static async getContents <T> (arrayOrPath : T[] | string, parentPath = "") : Promise<T[]> {
+    if(Array.isArray(arrayOrPath)) {
+      const contents = [];
+      for(const entry of arrayOrPath) {
+        contents.push(...(await this.getContent(entry, parentPath)));
+      }
+      return contents;
+    } else if(typeof arrayOrPath === "string") {
```
Comment (zelcion) [inline src/configuration/path-alias-utils.ts:13]: we don't need this second `if` here, neither the `else`.

**Review by zelcion — APPROVED** [review]
Just a small comment about the code but functionality is there :D

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
 # Changelog
+## 0.3.0
+### File Splitting
+- System `schemas`, `businessOperations`, and `protocols` configuration json can now be split into multiple files.
+- The allowed properties can be replaced by:
+  - A string path pointing to a json file of an array of the property type
+  - An array of paths pointing to a multiple files with multiple configurations
+  - A glob pattern that includes those configurations
+  - An array of glob patterns that include multiple configurations
+  - Or a mix of all the above. For more info consult our [documentation](https://mapikit.github.io/meta-system-docs/)
```
[the remark this answers is not in the document]
Comment (homemmakako) [reply CHANGELOG.md:10]: Yes, as soon as the actual page is done I'll update it.

File: src/configuration/path-alias-utils.ts (hunk 1 above)
Comment (homemmakako) [reply src/configuration/path-alias-utils.ts:13]: True, I'll change this right away.

**Pull request merged**